patreonmanager/admin.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib import admin
+from django.contrib import messages
+from django.core.urlresolvers import reverse
+from django.template.defaultfilters import linebreaksbr
+from django.utils.html import format_html
+from django.utils.translation import ugettext_lazy as _, ungettext
+
+from .models import Patron, Reward, Payment
+
+
+@admin.register(Patron)
+class PatronModelAdmin(admin.ModelAdmin):
```

Comment:
I think in the whole Django Girls website we use just PatronAdmin, without Model. Should we make it consistent?

This is a nitpick, sorry! ;)